docs/terminal/appearance.md

```diff
@@ -0,0 +1,132 @@
+
+### Powerline symbols and Nerd Fonts
+
+Powerline fonts are special patched fonts that contain additional characters that can be used in the terminal. VS Code's terminal [renders some of the powerline symbols without needing to configure a font](#_custom-glyphs), but if more glyphs are desired, configure a powerline font with the font family setting. Powerline fonts typically end in `" for Powerline"`, the following is an example of how to configure a DejaVu Sans Mono that has been patched:
+
+```json
+"editor.fontFamily": "'DejaVu Sans Mono for Powerline'"
+```
+
+Nerd Fonts work the same and typically have a `" NF"` suffix, the following is an example of how to configure Hack's nerd fonts variant:
+
+```json
+"terminal.integrated.fontFamily": "'Hack NF'"
+```
+
+## Terminal cursor
+
+TODO: ...
+
+## Customizing tabs
+
+### Visibility
+
+Terminal tabs appear on the right of the terminal view when there are two or more terminals by default, showing the active terminal in the view header when there is only one. This is designed to save horizontal space, but may not be desirable. How tabs are presented can be configured with the following settings:
+
+- `terminal.integrated.tabs.hideCondition`: When to hide the tabs to the right, set to `"never"` to always show them.
+- `terminal.integrated.tabs.showActiveTerminal`: When to show the active terminal in the terminal view header.
+- `terminal.integrated.tabs.showActions`: When to show the active terminal's actions in the view header.
+- `terminal.integrated.tabs.location`: Whether the tabs should be shown on the left or right of the terminal.
+- `terminal.integrated.tabs.enabled`: Whether to use tabs, disabling will show the original dropdown view.
+
+### Tab text
+
+The text on each tab is determined by the `terminal.integrated.tabs.title`, `terminal.integrated.tabs.description` and `terminal.integrated.tabs.separator` settings. By default, the title displays what the shell's detected process name.
+
+To change the title to display the escape sequence sent by the shell, use:
+
+```json
+"terminal.integrated.tabs.title": "${sequence}"
+```
+
+### Icons
+
+Each terminal has an associated icon which is determined by its [terminal profile](https://code.visualstudio.com/docs/terminal/profiles). The default icon and its color can be configured with the `terminal.integrated.tabs.defaultIcon` and `terminal.integrated.tabs.defaultColor` settings.
+
+### Status
+
+A tab's "status" is the icon that appears on the right of the tab. Some statuses such as those displays by tabs animate. If this is distracting the animation can be disabled with:
+
+```json
+"terminal.integrated.tabs.enableAnimation": false
+```
+
+### Visual bell
+
+The terminal features a visual bell which displays a yellow bell icon briefly when the terminal's bell is triggered. This can be disabled with `terminal.integrated.enableBell` and the duration can be configured with `terminal.integrated.bellDuration`.
```

Comment:
```suggestion
When the terminal's bell is triggered, a yellow bell icon is briefly shown. This can be disabled with `terminal.integrated.enableBell` and the duration can be configured with `terminal.integrated.bellDuration`.
```